pacta/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM 2dii/r-packages:latest
+LABEL maintainer='Mauro Lepore'
+
+COPY PACTA_analysis /pacta/PACTA_analysis
+COPY create_interactive_report /pacta/create_interactive_report
+COPY pacta-data /pacta/pacta-data
+COPY StressTestingModelDev /pacta/StressTestingModelDev
```

Comment:
does the `/pacta` in these lines mean that your local clones of these repos need to be in a directory at the root level of your drive named "pacta", or that the repos will be copied into the docker image into a directory at the root level of the docker drive named "pacta", or both, out neither?

Reply:
I guess better stated... what does the `/pacta` mean in these lines?